kubernetes-server-mock/src/main/java/io/fabric8/kubernetes/client/server/mock/KubernetesMockServer.java

```diff
@@ -29,6 +29,7 @@
 import okhttp3.mockwebserver.Dispatcher;
 import okhttp3.mockwebserver.MockWebServer;
 
+import java.net.*;
```

Comment:
nit, we usually avoid adding wildcard imports in codebase since they become difficult to deal with during model upgrades.